lmp/build.sh

```diff
@@ -46,6 +46,9 @@ su builder -c $HERE/bb-config.sh
 touch ${archive}/customize-target.log && chown builder ${archive}/customize-target.log
 touch ${archive}/bitbake_debug.log ${archive}/bitbake_warning.log && chown builder ${archive}/bitbake_*.log
 touch ${archive}/bitbake_global_env.txt ${archive}/bitbake_image_env.txt && chown builder ${archive}/bitbake_*_env.txt
+touch "${archive}/app-preload.log" && chown builder "${archive}/app-preload.log"
+# Container preloading tools store their context in this directory
+mkdir /run/containers && chown builder /run/containers
```

Comment:
this can't be problematic but needs root privileges

Reply:
At this point I believe the script is running as root.

Reply:
ok

Reply:
Although it works (CI worker run this script in a privileged container and as a root user), I don't like it too, and I'm looking for a way to avoid it so the preloading script is self-contained.

Reply:
I think I fixed it, see the latest version of the PR.